src/shared/extensionUtilities.ts

```diff
@@ -168,24 +167,35 @@ export function setMostRecentVersion(context: vscode.ExtensionContext): void {
 /**
  * Publishes a toast with a link to the welcome page
  */
-export async function promptWelcome(): Promise<void> {
+async function promptQuickStart(): Promise<void> {
     const view = localize(
-        'AWS.message.prompt.welcome.openPage',
+        'AWS.command.quickStart',
         'View Welcome Page'
     )
     const prompt = await vscode.window.showInformationMessage(
         localize(
-            'AWS.message.prompt.welcome.toastMessage',
+            'AWS.message.prompt.quickStart.toastMessage',
             'You are now using the AWS Toolkit for Visual Studio Code, version {0}',
             pluginVersion
         ),
-        view,
-        localize(
-            'AWS.message.prompt.welcome.dismiss',
-            'Dismiss'
-        ),
+        view
     )
     if (prompt === view) {
-        vscode.commands.executeCommand('aws.welcome')
+        vscode.commands.executeCommand('aws.quickStart')
+    }
+}
+
+/**
+ * Checks if a user is new to this version
+ * If so, pops a toast with a link to a quick start page
+ *
+ * @param context VS Code Extension Context
+ */
+export function toastNewUser(context: vscode.ExtensionContext): void {
```

Comment:
This method should have a try/catch that eats and logs the error. Otherwise we risk breaking extension activation.

Reply:
✔️